metron-sensors/bro-plugin-kafka/README.md

```diff
@@ -43,6 +43,7 @@ The following examples highlight different ways that the plugin can be used.  Si
 The goal in this example is to send all HTTP and DNS records to a Kafka topic named `bro`. 
  * Any configuration value accepted by librdkafka can be added to the `kafka_conf` configuration table.  
  * By defining `topic_name` all records will be sent to the same Kafka topic.
+ * By providing a set of logs via `logs_to_send`.
```

Comment:
This doesn't sound like a complete thought to me.  Maybe this?

"Defining `logs_to_send` will ensure that only HTTP and DNS records are sent."